IS7024Project/Pages/Index.cshtml.cs

```diff
@@ -28,17 +29,16 @@ public void OnGet()
             {
                 //Consuming Parks data
                 string parksJSON = webClient.DownloadString("https://raw.githubusercontent.com/JMFrank215/IS7024Project/master/Parks_Data.txt");
-                var park = Parks.FromJson(parksJSON);
-                ViewData["Parks"] = park;
+                var parks = Parks.FromJson(parksJSON);
+                ViewData["Parks"] = parks;
                 QuickTypeParks.Parks[] Parksparks = QuickTypeParks.Parks.FromJson(parksJSON);
 
 
                 //Consuming Crime data
-                string jsonString = webClient.DownloadString("https://raw.githubusercontent.com/JMFrank215/IS7024Project/master/PDI_Crime_Data.txt");
-                var crime = Crime.FromJson(jsonString);
-                ViewData["Crime"] = crime;
+                string crimesJSON = webClient.DownloadString("https://raw.githubusercontent.com/JMFrank215/IS7024Project/master/PDI_Crime_Data.txt");
+                var crimes = Crime.FromJson(crimesJSON);
```

Comment:
Again, changed crimes from crime to better represent you're working with an array of multiple crimes.